DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -360,6 +355,92 @@ private int GetMetadataSheetId()
             return metadataSheet.Properties.SheetId.Value;
         }
 
+        private CellData MakeCellData(double number)
+        {
+            return new CellData()
+            {
+                UserEnteredValue = new ExtendedValue() { NumberValue = largestKey }
+            };
+        }
+
+        private Request MakeAddMetadataRequest(int eventKey, string name, string description, DateTime time)
+        {
+            var appendCellsRequest = new AppendCellsRequest()
+            {
+                Rows = new[]
+                {
+                    new RowData()
+                    {
+                        Values = new List<CellData>()
+                        {
+                            MakeCellData(eventKey),
+                            MakeCellData(name),
+                            MakeCellData(description),
+                            MakeCellData(time.ToString("s")),
+                            MakeCellData("Europe/London")
+                        }
+                    }
+                },
+                SheetId = metadataSheetId,
+                Fields = "*"
+            };
+
+            return new Request() { AppendCells = appendCellsRequest };
+        }
+
+        private Request MakeAddResponseSheetRequest(int eventKey)
+        {
+            var addResponseSheet = new AddSheetRequest()
+            {
+                Properties = new SheetProperties()
+                {
+                    Title = eventKey.ToString(),
+                    SheetId = eventKey
+                }
+            };
+
+            return new Request() { AddSheet = addResponseSheet };
+        }
+
+        private Request MakeAddResponseColumnsRequest(List<EventResponse> responseList)
+        {
+            var addResponseColumns = new AppendCellsRequest()
+            {
+                Rows = new[]
+                {
+                    new RowData()
+                    {
+                        Values = responseList
+                            .Select(response => response.Emoji)
+                            .Prepend("User id / Response emoji")
+                            .Select(MakeCellData)
+                            .ToList()
+                    },
+                    new RowData()
+                    {
+                        Values = responseList
+                            .Select(response => response.ResponseName)
+                            .Prepend("Response name:")
+                            .Select(MakeCellData)
+                            .ToList()
+                    }
+                },
+                SheetId = largestKey,
+                Fields = "*"
+            };
+
+            return new Request() { AppendCells = addResponseColumns };
+        }
+
+        private CellData MakeCellData(string stringValue)
```

Comment:
I'd recommend defining this near the other method with the same name 🙂

Reply:
Yep, sorry these got split up! Moved in c3e7c4b 🙂